target/linux/lantiq/base-files/etc/uci-defaults/80_wifi_setup

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+
+. /lib/functions/uci-defaults.sh
+. /lib/functions/system.sh
+. /lib/functions/lantiq.sh
+
+board=$(board_name)
+
+case "$board" in
+arcadyan,vgv952cjw33-e-ir-vpe|arcadyan,vgv952cjw33-e-ir-smp)
```

Comment:
This file looks like some kind of custom configuration of yours. I haven't seen something like that before.

Is it really needed for the default image? Can you show a similar example already in the trunk?